content/general/push/admin.textile

```diff
@@ -230,7 +230,7 @@ bq(definition).
   default: list(Object params, callback("ErrorInfo":/realtime/types#error-info err, "PaginatedResult":#paginated-result<"DeviceDetails":#device-details device> resultPage))
   ruby:    "Deferrable":/realtime/types#deferrable list(Hash params) -> yields "PaginatedResult":#paginated-result<"DeviceDetails":#device-details>
   python: "PaginatedResult":#paginated-result list(params=Object)
-  php: "PaginatedResult":#paginated-result list(Hash params)
+  php: "PaginatedResult":#paginated-result list_ (array params)
```

Comment:
Why is there a space here?  Should it not be `list_(array params)` for consistency?